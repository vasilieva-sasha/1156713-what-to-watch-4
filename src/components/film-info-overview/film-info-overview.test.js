import React from "react";
import renderer from "react-test-renderer";
import FilmInfoOverview from "./film-info-overview";

const mock = {
  rating: {
    score: 8,
    level: ` `,
    count: 200
  },
  text: `text`,
  director: `director`,
  actors: [`actor`, `actor`]
};

describe(`FilmOverviewComponent`, () => {
  it(`Overview renders correctly`, () => {
    const tree = renderer.create(
        <FilmInfoOverview film={mock} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
