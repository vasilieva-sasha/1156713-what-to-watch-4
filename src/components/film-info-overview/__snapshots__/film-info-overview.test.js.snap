// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmOverviewComponent Overview renders correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      8
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Good
      </span>
      <span
        className="movie-rating__count"
      >
        200
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      text
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        director
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        actor
        actor
         and other
      </strong>
    </p>
  </div>,
]
`;
